arm-analysisservices/2016-05-16/swagger/analysisservices.json

```diff
@@ -525,6 +528,28 @@
           "description": "An array of administrator user identities."
         }
       }
+    },
+    "BackupConfiguration": {
+      "description": "An object that represents backup configurations",
+      "type": "object",
+      "properties": {
+        "storageAccount": {
+          "type": "string",
+          "description": "The name of storage account for backup configuration. It is in the format of 'Resource Group Name/Storage Account Name'"
```

Comment:
@johanste  Understand. We already met with Azure API review board and got exception on this change. We agreed that we will not bump the version before GA.